SQF/dayz_code/stringtable.xml

```diff
@@ -13403,6 +13412,24 @@
 			<French>Votre humanité est trop %1 ce commerçant refuse de vous parler.</French>
 			<Czech>Tvá lidskost je příliš %1, tento obchodník s tebou odmítá mluvit.</Czech>
 		</Key>
+		<Key ID="STR_EPOCH_ACTIONS_HUMANITY_HIGH">
+			<English>high</English>
+			<German>high</German> -->
```

Comment:
I agree, for sentences and longer phrases it is usually very bad, but for single words it's usually okay.

It's fine to leave them all as English too, because at least then the formatting is done and it will be easy for a native speaker to go back and correct the stringtable later.